File: api/src/main/java/com/ricky/adocao/dto/UsuarioDTO.java
package com.ricky.adocao.dto;

import com.ricky.adocao.enums.RolesEnum;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UsuarioDTO {
    private Long id;
    private String login;
    private RolesEnum role;
    private String telefone;
    private String email;
}

File: src/main/kotlin/com/ricky/adocao/dto/UsuarioDTO.kt
package com.ricky.adocao.dto

import jakarta.validation.constraints.NotBlank

data class UsuarioDTO(
    var id: String = "",
    @field:NotBlank(message = "{nome.obrigatorio}")
    var nome: String = "",
    @field:NotBlank(message = "{login.obrigatorio}")
    var login: String = "",
    @field:NotBlank(message = "{senha.obrigatorio}")
    var senha:String = "",
    @field:NotBlank(message = "{email.obrigatorio}")
    var email: String = "",
    @field:NotBlank(message = "{telefone.obrigatorio}")
    var telefone: String = "",
)
